torch/testing/_core.py

```diff
@@ -9,31 +9,13 @@
 from typing import cast, List, Optional, Tuple, Union
 import operator
 
+from ._dtype_getters import get_all_dtypes, get_all_complex_dtypes
+
```

Comment:
We need these here, because `is_integral` uses them. This will not be a problem in the future, since `is_integral` is only used by `_compare_tensors_internal`, which in turn is only (indirectly) used by `torch.testing._internal.common_utils.TestCase.assertEqual`, which ultimately will be replaced by `torch.testing.assert_close` (#58981).

Reply:
Would you add a brief comment with this explanation and a TODO to remove this?